Documentation/ceph-cluster-crd.md

```diff
@@ -433,6 +433,9 @@ The following are the settings for Storage Class Device Sets which can be config
 
   However, if there are more OSDs than nodes, this anti-affinity will not be effective. Another placement scheme to consider is to add labels to the nodes in such a way that the OSDs can be grouped on those nodes, create multiple storageClassDeviceSets, and add node affinity to each of the device sets that will place the OSDs in those sets of nodes.
 
+  Rook will automatically add required nodeAffinity to the OSD daemons to match the topology labels that are found in the OSD prepare jobs. To ensure data durability,
+  the OSDs are required to run in the same topology that the Ceph CRUSH map expects. See the [OSD Topology](#osd-topology) section for the related labels.
+
```

Comment:
I think an example would be good here. I can see users not fully understanding what is meant by this.

Reply:
Also, is this statement correct?

> Rook will automatically add required nodeAffinity to the OSD daemons to match the topology labels that are found in the OSD prepare jobs.

I thought it applies topology labels found on the node on which the OSD prepare job is run?

Reply:
I'll clarify